### ISSUE TITLE
Potential Conflict: flutter_sound (9.17.0) and flutter_secure_storage

### ISSUE THREAD

**Issue by yanaico** [Canardoux/flutter_sound#1127]
I'm encountering potential conflicts between the flutter_sound (version 9.17.0) and flutter_secure_storage (version 9.2.2) packages in my Flutter application.

Can you please fix this issue?

**Comment by Larpoux** [comment]
What is exactly the error that you have ? What is the conflict ?

**Comment by yanaico** [comment]
> What is exactly the error that you have ? What is the conflict ?

Because no versions of flutter_secure_storage match >9.2.2 <10.0.0 and flutter_secure_storage 9.2.2 depends on flutter_secure_storage_web ^1.2.1, flutter_secure_storage ^9.2.2 requires flutter_secure_storage_web ^1.2.1.
And because no versions of flutter_secure_storage_web match >1.2.1 <2.0.0 and flutter_secure_storage_web 1.2.1 depends on js ^0.6.3, flutter_secure_storage ^9.2.2 requires js ^0.6.3.
And because flutter_sound >=9.17.0 depends on flutter_sound_web 9.17.0 which depends on js ^0.7.1, flutter_secure_storage ^9.2.2 is incompatible with flutter_sound >=9.17.0.
So, because be_pass depends on both flutter_sound ^9.17.0 and flutter_secure_storage ^9.2.2, version solving failed.


You can try one of the following suggestions to make the pubspec resolve:
* Consider downgrading your constraint on flutter_secure_storage: flutter pub add flutter_secure_storage:^4.2.1
* Consider downgrading your constraint on flutter_sound: flutter pub add flutter_sound:^9.6.0

**Comment by Larpoux** [comment]
> flutter_secure_storage_web 1.2.1 depends on js ^0.6.3, flutter_secure_storage ^9.2.2 requires js ^0.6.3.
> And because flutter_sound >=9.17.0 depends on flutter_sound_web 9.17.0 which depends on js ^0.7.1,

I am not sure that it would be a good idea to downgrade the flutter sound dependency.
Is there a reason why flutter_secure_storage_web depends on  js ^0.6.3, on not  js ^0.7.1 ?
Could the flutter_secure_storage_web guys upgrade their dependency to js ^0.7.1 ?

**Comment by yanaico** [comment]
Correct, I dont want to downgrade, the question is where is the outdated issue?
flutter_sound or the flutter_secure_storage?

Because if Im commit one of them it works fine with the latest version.

**Comment by Larpoux** [comment]
> flutter_secure_storage_web 1.2.1 depends on js ^0.6.3, flutter_secure_storage ^9.2.2 requires js ^0.6.3.
> And because flutter_sound >=9.17.0 depends on flutter_sound_web 9.17.0 which depends on js ^0.7.1,

 The problem is with flutter_secure_storage_web 1.2.1.
BTW : I am currently working on a future Flutter Sound 10.0.0 This version will not depends anymore on js, so we will have less problems.. But this new version will not be tomorrow! My dev are very slow!
